Cakes-Co/style.css

```diff
@@ -1 +1,245 @@
-/*  Add your styling here */
+* {
+    margin: 0;
+    padding: 0;
+  }
+  body {
+    font-family: "Roboto", sans-serif;
+    font-size: 4vw;
+    background-image: linear-gradient(rgb(243, 207, 232), rgb(247, 243, 168), rgb(255, 126, 203));
+     
+  }
+  h1 {
+    font-size: 2.4em;
+    font-weight: 500;
+  }
+  h4 {
+    font-size: 1.2em;
+    font-weight: 400;
+    font-style: italic;
+  }
+
+  h6 {
+    font-size: 0.7em;
+    font-weight: 200;
+  }
+  p {
+    font-size: 1em;
+    font-weight: 300;
+    line-height: 1.4em;
+  }
+  img {
+    width: 100%;
+    border-radius: 10%;
+  }
+  
+  :root {
+    
+    --backgroundColor: linear-gradient(rgb(249, 170, 217), rgb(246, 239, 115), rgb(243, 129, 207));
+    --generalPadding: 1em;
+    --sectionGap: 2em;
+    --white: rgb(75, 6, 52);
+  }
+  
+  header {
+    height: 40vw;
+    padding: var(--generalPadding);
+    background-color: var(--backgroundColor);
```

Comment:
This won't work - gradiants are background images so setting them as the background color won't work. The background-image above does all the work of setting the gradiant, so you can remove this.